portfolio/src/main/webapp/index.html

```diff
@@ -8,9 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>Data</h1>
-      <p>Click here to see some data:</p>
-      <button onclick="getDataUsingAsyncAwait()">Data</button>
+
+      <h1>Add a comment</h1>
+      <form action="/data" method="POST">
+      <p>Add comment:</p>
+      <textarea name="comment"></textarea>
+      <br/><br/>
+      <input type="submit" />
+      </form>
+
+     
+      <p>Click here to see all comments:</p>
+      <button onclick="getDataUsingAsyncAwait()">comments</button>
```

Comment:
Just a note: this approach is totally fine and works, and if you wanted to be able to see the comments immediately without clicking a button, you can add `onload="getDataUsingAsyncAwait()"` to data-container below; see: https://www.w3schools.com/jsref/event_onload.asp

Reply:
mm okay, gotcha